### ISSUE TITLE
Can't install netdata on Aliyun CentOS 6.8

### ISSUE THREAD

**Issue by startryst** [netdata/netdata#7608]
##### Bug report summary
After run `bash <(curl -Ss https://my-netdata.io/kickstart.sh)`, failed with below error:
`configure: error: libuv required but not found. Try installing 'libuv1-dev' or 'libuv-devel'.`

but libuv-devel already installed
```
[root@alyhk5 ~]# yum install libuv-devel
Loaded plugins: fastestmirror
Setting up Install Process
Loading mirror speeds from cached hostfile
 * elrepo: mirror-hk.koddos.net
 * epel: mirrors.aliyun.com
Package 1:libuv-devel-0.10.34-1.el6.x86_64 already installed and latest version
Nothing to do
```
##### OS / Environment
[root@alyhk5 ~]# uname -a
Linux alyhk5 4.18.11-1.el6.elrepo.x86_64 #1 SMP Sat Sep 29 10:43:38 EDT 2018 x86_64 x86_64 x86_64 GNU/Linux
[root@alyhk5 ~]# cat /etc/redhat-release 
CentOS release 6.8 (Final)
[root@alyhk5 ~]# yum repolist
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * elrepo: mirror-hk.koddos.net
 * epel: mirrors.aliyun.com
repo id                                                                repo name                                                                                                              status
base                                                                   CentOS-6                                                                                                                6,713
elrepo                                                                 ELRepo.org Community Enterprise Linux Repository - el6                                                                    258
epel                                                                   Extra Packages for Enterprise Linux 6 - x86_64                                                                         12,582
extras                                                                 CentOS-6                                                                                                                   47
updates                                                                CentOS-6                                                                                                                  781
repolist: 20,381

**Comment by thiagoftsm** [comment]
Hi @startryst ,

I remember months ago I had a problem with a Linux distribution that probably has relationship with this error you are reporting, Netdata needs the version `1.x` of libuv to run, it is possible that the package manager installed the version `0.x`, please, could you check the version that you have installed?

best regards!

**Comment by startryst** [comment]
> Hi @startryst ,
> 
> I remember months ago I had a problem with a Linux distribution that probably has relationship with this error you are reporting, Netdata needs the version `1.x` of libuv to run, it is possible that the package manager installed the version `0.x`, please, could you check the version that you have installed?
> 
> best regards!

Yes, it's 0.10, any easy way to get upgraded?

```
yum install libuv-devel
Loaded plugins: fastestmirror
Setting up Install Process
Loading mirror speeds from cached hostfile
 * elrepo: mirror-hk.koddos.net
 * epel: mirrors.aliyun.com
Package 1:libuv-devel-0.10.34-1.el6.x86_64 already installed and latest version
Nothing to do
```

**Comment by Ferroin** [comment]
@startryst You should be able to get things working by adding the [OKay repository](https://okay.network/blog-news/rpm-repositories-for-centos-6-and-7.html). They have packages for libuv 1.x for CentOS 6.

**Comment by startryst** [comment]
> @startryst You should be able to get things working by adding the [OKay repository](https://okay.network/blog-news/rpm-repositories-for-centos-6-and-7.html). They have packages for libuv 1.x for CentOS 6.

Let me have a try....

**Comment by startryst** [comment]
@Ferroin Great, issue fixed, after libuv 1.9.1-4 installed, netdata installed smoothly, Thanks.

@thiagoftsm Thanks for your advice as well.
